src/Microsoft.AspNet.Mvc.Abstractions/ModelBinding/UnsupportedContentTypeException.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.AspNet.Mvc.Formatters;
+
+namespace Microsoft.AspNet.Mvc.ModelBinding
+{
+    /// <summary>
+    /// The <see cref="Exception"/> that is added to model state when a model binder for the body of the request is
+    /// unable to understand the request content type header.
+    /// </summary>
+    public class UnsupportedContentTypeException : Exception
```

Comment:
No. Move these out. I just missed it in my review. `TooManyModelErrorsException` is only here because the code that throws it is in this assembly. It's a bit of a wart.

:trollface: Please ask for permission before putting any new code in `.Abstractions` :laughing: